File: Iniciante/URI 1044.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    val a: Int = leitor.nextInt();
    val b: Int = leitor.nextInt();
    if (a % b == 0 || b % a == 0) {
        println("Sao Multiplos");
    } else {
        println("Nao sao Multiplos");
    }
}

File: Iniciante/URI 1044.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val a = leitor.nextInt()
        val b = leitor.nextInt()
        if (a % b == 0 || b % a == 0) {
            println("Sao Multiplos")
        } else {
            println("Nao sao Multiplos")
        }
    }
    
}